packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -497,6 +527,10 @@ export class ConnectionFlow {
     const trigger = await fetchTrigger(this.client, this.trigger._id)
     logger.debug(`Refetched trigger`, trigger)
     this.trigger = trigger
+
+    this.watchCurrentJobIfTriggerIsAlreadyRunning()
```

Comment:
Do we really want to only `watchJob` if the trigger is running? Do we need to subscribe here to the realtime? Don't we have already subscribe in a way or other?